### DISCUSSION TITLE
钓鱼佬养成记 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#467] [General]
# 钓鱼佬养成记 - Miao Yu | 于淼

https://yufree.cn/cn/2024/06/15/angler/

**Comment by earfanfan** [comment]
哇，三水啊，怎么大家都是沉默看看不吐槽，都在心里吐槽了吧。

话说在美国钓鱼只能这么钓嘛，你这一看就是小时候没钓过鱼的。我记得我们那小时候钓小龙虾是没得这么多装备的（PS当然我是在村子里），通常是地上随便捡一根看得顺眼的麻杆，不知道从哪里弄来的粗点的白棉线，重点是要找几块砖头，最好是在那种地湿点的地方放着的，一看就已经放了很久的砖头，翻开砖头，一般砖头下面会有好多条蚯蚓，把蚯蚓捡出来，扔到事先准备的草木灰堆里（PS通常是选的砖头离谁家近就从谁家厨房灶里扒灰），用线系几条肥蚯蚓，要打死结系紧，然后提起线把蚯蚓们摔到将死状态，再把线的另一头系在麻杆上，就可以去钓小龙虾了。当然，钓的位置也是事先看好的，通常是小伙伴们已经先知道哪里有龙虾洞了。

**Reply by yufree** [reply]
用活饵确实容易上鱼，毕竟国内还有泥鳅钓犯法的新闻，不过钓鱼主要就是图一乐，上鱼率低反而更有意思

**Comment by iotanano** [comment]
水看起来有点浅？如果深一点并追求效率，可以考虑下台钓。之前朋友怀孕时一起出去钓鱼，基本就是围观孕妇朋友重复拿鱼竿-取鱼-放回鱼竿的动作，效率惊人。

PS：路亚貌似更适合河流（钓翘嘴之类的），池塘里面钓容易钓到螺蛳。

**Reply by earfanfan** [reply]
不好意思，我想插嘴问一句，如果高效率地钓到了很多鱼的话，鱼最后是怎么处理了呢？把最肥美的煮鱼汤吃掉，剩下的放回去么

**Reply by yufree** [reply]
美国这边主要是运动钓鱼，路亚算是主流，当然鲤鱼鲶鱼还是用饵钓，这边主要是抓了放，法律上有些鱼抓了必须放或者额外买证才能带走，我住处附近的钓鱼佬为了怕麻烦都不带网，当然我不带单纯是因为菜，带了也用不上。

**Comment by YuanyeChi** [comment]
永不空军 Release aquatic plants。水草清除者，哈哈哈哈哈🤣
